Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -152,7 +152,9 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
     connect(m_ui->m_assetBrowserTreeViewWidget, &AzAssetBrowser::AssetBrowserTreeView::selectionChangedSignal,
         this, &AzAssetBrowserWindow::SelectionChangedSlot);
 
-    connect(m_ui->m_assetBrowserTreeViewWidget, &QAbstractItemView::doubleClicked, this, &AzAssetBrowserWindow::DoubleClickedItem);
```

Comment:
was this doubleClicked not firing at all, or was it in addition to the activated signal? Could you comment this please?

Reply:
Totally rewritten.